### ISSUE TITLE
Why accese key not use in .env ?

### ISSUE THREAD

**Issue by 2er7** [aws/aws-sdk-php-laravel#136]
Why accese key is stored in config and not in .env ?

**Comment by qasimalbaqali** [comment]
Wouldn't this just do what you need? 

```
'credentials' => [
        'key'    => env('AWS_ACCESS_KEY_ID', ''),
        'secret' => env('AWS_SECRET_ACCESS_KEY', ''),
],
```

**Comment by howardlopez** [comment]
Credentials have been added to the published service provider config by default in #143, and are available in [release 3.3.0](https://github.com/aws/aws-sdk-php-laravel/releases/tag/3.3.0). This won't override an existing configuration, but any project publishing for the first time will have these credentials included and pulling from the `.env` file by default. Thanks for the feedback!
